pymongo/common.py

```diff
@@ -585,18 +584,11 @@ def validate_tzinfo(dummy, value):
 
 
 # Dictionary where keys are the names of public URI options, and values
-# are lists of aliases for that option. Aliases of option names are assumed
-# to have been deprecated.
+# are lists of aliases for that option.
 URI_OPTIONS_ALIAS_MAP = {
     'journal': ['j'],
     'wtimeoutms': ['wtimeout'],
     'tls': ['ssl'],
```

Comment:
Should we remove ssl here (and INTERNAL_URI_OPTION_NAME_MAP) as well?

Reply:
I think the decision was made to keep `ssl` indefinitely since we never deprecated it? Do you think we should deprecate it in the next 3.x release and remove?

Reply:
Note also that the follow up PR to this will be a better place remove this kind of bookkeeping code as that PR will get rid of all remaining options that have been deprecated in 3.x

Reply:
> Do you think we should deprecate it in the next 3.x release and remove?

I think you're right that we should keep it. The URI-options spec still has `ssl` and doesn't say to remove it. It might be helpful to ask other drivers what their plan is.